multisine:
  test_namespace:
  - "deflection_6dof_jnt1_test1_"
  - "deflection_6dof_jnt1_test2_"
  - "deflection_6dof_jnt1_test3_"
  - "deflection_6dof_jnt1_test4_"
  - "deflection_6dof_jnt1_test5_"
  - "deflection_6dof_jnt1_test6_"
  - "deflection_6dof_jnt1_test7_"
deflection_6dof_jnt1_test1_:
  signal_ffw: 0
  topic_name: "/rigid/joint_feedforward"
  test_duration: 100.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-01, 1.25663706e+00, 1.88495559e+00, 3.76991118e+00, 5.02654825e+00, 1.00530965e+01, 1.13097336e+01, 1.25663706e+01, 1.38230077e+01, 1.50796447e+01, 1.88495559e+01, 1.94778745e+01, 2.01061930e+01, 2.19911486e+01, 2.32477856e+01, 2.38761042e+01, 2.70176968e+01, 3.39292007e+01, 3.58141563e+01, 4.02123860e+01, 4.20973416e+01, 4.27256601e+01, 4.46106157e+01, 4.58672527e+01, 4.64955713e+01, 5.02654825e+01, 5.15221195e+01, 5.65486678e+01, 5.71769863e+01, 6.28318531e+01 ]

  sinusoidal_coefficients: 
  - [-1.29376268e+00, 0.00000000e+00 ]
  - [-1.76850317e+00, 0.00000000e+00 ]
  - [-5.75447653e-01, 0.00000000e+00 ]
  - [-1.62092768e+00, 0.00000000e+00 ]
  - [-1.75099508e+00, 0.00000000e+00 ]
  - [1.36666172e+00, 0.00000000e+00 ]
  - [8.92713562e-01, 0.00000000e+00 ]
  - [1.16428188e+00, 0.00000000e+00 ]
  - [1.31552923e+00, 0.00000000e+00 ]
  - [-3.87284541e-01, 0.00000000e+00 ]
  - [1.36676394e+00, 0.00000000e+00 ]
  - [-7.60825632e-01, 0.00000000e+00 ]
  - [1.01885314e+00, 0.00000000e+00 ]
  - [1.24547991e+00, 0.00000000e+00 ]
  - [1.28422953e-01, 0.00000000e+00 ]
  - [-8.40482738e-01, 0.00000000e+00 ]
  - [1.34624228e+00, 0.00000000e+00 ]
  - [1.41510326e+00, 0.00000000e+00 ]
  - [1.55578963e+00, 0.00000000e+00 ]
  - [5.01251645e-01, 0.00000000e+00 ]
  - [9.46738978e-02, 0.00000000e+00 ]
  - [9.78303449e-01, 0.00000000e+00 ]
  - [1.62959342e+00, 0.00000000e+00 ]
  - [-1.28329302e+00, 0.00000000e+00 ]
  - [-5.43058322e-01, 0.00000000e+00 ]
  - [-1.37458933e+00, 0.00000000e+00 ]
  - [5.88507881e-01, 0.00000000e+00 ]
  - [-1.24492577e-01, 0.00000000e+00 ]
  - [1.42488364e+00, 0.00000000e+00 ]
  - [6.75349765e-01, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [-5.23144895e+00, 0.00000000e+00 ]
  - [-3.15074732e-01, 0.00000000e+00 ]
  - [-1.70168609e+00, 0.00000000e+00 ]
  - [7.74253842e-01, 0.00000000e+00 ]
  - [4.01113199e-01, 0.00000000e+00 ]
  - [-1.16580929e+00, 0.00000000e+00 ]
  - [-1.55882586e+00, 0.00000000e+00 ]
  - [-1.36796318e+00, 0.00000000e+00 ]
  - [1.22321642e+00, 0.00000000e+00 ]
  - [-1.75410554e+00, 0.00000000e+00 ]
  - [1.16568945e+00, 0.00000000e+00 ]
  - [-1.62727377e+00, 0.00000000e+00 ]
  - [-1.47946404e+00, 0.00000000e+00 ]
  - [-1.29447107e+00, 0.00000000e+00 ]
  - [1.79175420e+00, 0.00000000e+00 ]
  - [-1.58759703e+00, 0.00000000e+00 ]
  - [-1.18933060e+00, 0.00000000e+00 ]
  - [-1.10650726e+00, 0.00000000e+00 ]
  - [8.97994532e-01, 0.00000000e+00 ]
  - [1.72499923e+00, 0.00000000e+00 ]
  - [-1.79385407e+00, 0.00000000e+00 ]
  - [1.50658485e+00, 0.00000000e+00 ]
  - [-7.55844464e-01, 0.00000000e+00 ]
  - [-1.25699426e+00, 0.00000000e+00 ]
  - [1.71229764e+00, 0.00000000e+00 ]
  - [-1.15645135e+00, 0.00000000e+00 ]
  - [-1.69721361e+00, 0.00000000e+00 ]
  - [1.79203157e+00, 0.00000000e+00 ]
  - [-1.09388399e+00, 0.00000000e+00 ]
  - [1.66456548e+00, 0.00000000e+00 ]

deflection_6dof_jnt1_test2_:
  signal_ffw: 0
  topic_name: "/rigid/joint_feedforward"
  test_duration: 100.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-01, 3.39292007e+01, 3.58141563e+01, 4.14690230e+01, 4.27256601e+01, 4.71238898e+01, 4.83805269e+01, 5.02654825e+01, 5.08938010e+01, 5.27787566e+01, 5.34070751e+01, 5.46637122e+01, 5.65486678e+01, 5.96902604e+01, 6.15752160e+01, 6.28318531e+01, 6.34601716e+01, 6.40884901e+01, 6.47168087e+01, 6.59734457e+01, 7.09999940e+01, 7.66548607e+01, 7.79114978e+01, 7.91681349e+01, 8.41946831e+01, 8.48230016e+01, 8.92212314e+01, 9.11061870e+01, 9.36194611e+01, 9.42477796e+01 ]

  sinusoidal_coefficients: 
  - [-3.84204173e+00, 0.00000000e+00 ]
  - [-1.58128742e+00, 0.00000000e+00 ]
  - [4.81909586e-01, 0.00000000e+00 ]
  - [8.76499233e-01, 0.00000000e+00 ]
  - [-1.03433423e-01, 0.00000000e+00 ]
  - [4.23904068e-01, 0.00000000e+00 ]
  - [-1.72956373e+00, 0.00000000e+00 ]
  - [-8.99793322e-01, 0.00000000e+00 ]
  - [-4.66392418e-01, 0.00000000e+00 ]
  - [1.01926254e+00, 0.00000000e+00 ]
  - [-1.48288422e-01, 0.00000000e+00 ]
  - [-1.66831152e+00, 0.00000000e+00 ]
  - [1.05809286e+00, 0.00000000e+00 ]
  - [1.46632440e+00, 0.00000000e+00 ]
  - [-7.32116489e-01, 0.00000000e+00 ]
  - [-9.93002814e-01, 0.00000000e+00 ]
  - [-1.56329989e+00, 0.00000000e+00 ]
  - [-7.20015424e-02, 0.00000000e+00 ]
  - [1.44706281e+00, 0.00000000e+00 ]
  - [6.75887372e-03, 0.00000000e+00 ]
  - [6.68366488e-01, 0.00000000e+00 ]
  - [3.67663285e-01, 0.00000000e+00 ]
  - [-1.38806798e-01, 0.00000000e+00 ]
  - [4.25959731e-01, 0.00000000e+00 ]
  - [1.28790778e+00, 0.00000000e+00 ]
  - [1.51804178e+00, 0.00000000e+00 ]
  - [-1.28728261e+00, 0.00000000e+00 ]
  - [-1.32195810e+00, 0.00000000e+00 ]
  - [-1.12847803e+00, 0.00000000e+00 ]
  - [-1.72459451e+00, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [3.49877722e+00, 0.00000000e+00 ]
  - [7.06988717e-01, 0.00000000e+00 ]
  - [1.66375062e+00, 0.00000000e+00 ]
  - [-1.49400537e+00, 0.00000000e+00 ]
  - [-1.72904728e+00, 0.00000000e+00 ]
  - [-1.67946667e+00, 0.00000000e+00 ]
  - [9.44048948e-02, 0.00000000e+00 ]
  - [1.48009288e+00, 0.00000000e+00 ]
  - [-1.66816698e+00, 0.00000000e+00 ]
  - [1.40050236e+00, 0.00000000e+00 ]
  - [1.72577910e+00, 0.00000000e+00 ]
  - [-4.65875141e-01, 0.00000000e+00 ]
  - [1.37140164e+00, 0.00000000e+00 ]
  - [-9.22060578e-01, 0.00000000e+00 ]
  - [1.56981158e+00, 0.00000000e+00 ]
  - [1.41924218e+00, 0.00000000e+00 ]
  - [-7.45919845e-01, 0.00000000e+00 ]
  - [-1.73064114e+00, 0.00000000e+00 ]
  - [9.52004294e-01, 0.00000000e+00 ]
  - [-1.73212508e+00, 0.00000000e+00 ]
  - [-1.59799537e+00, 0.00000000e+00 ]
  - [1.69266851e+00, 0.00000000e+00 ]
  - [-1.72656759e+00, 0.00000000e+00 ]
  - [-1.67894648e+00, 0.00000000e+00 ]
  - [1.15827308e+00, 0.00000000e+00 ]
  - [-8.34177512e-01, 0.00000000e+00 ]
  - [-1.15896783e+00, 0.00000000e+00 ]
  - [1.11925411e+00, 0.00000000e+00 ]
  - [-1.31409296e+00, 0.00000000e+00 ]
  - [-1.61482950e-01, 0.00000000e+00 ]

deflection_6dof_jnt1_test3_:
  signal_ffw: 0
  topic_name: "/rigid/joint_feedforward"
  test_duration: 100.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-01, 6.40884901e+01, 6.78584013e+01, 6.84867198e+01, 6.97433569e+01, 7.09999940e+01, 7.35132681e+01, 7.47699052e+01, 7.72831793e+01, 7.91681349e+01, 8.10530905e+01, 8.16814090e+01, 8.54513202e+01, 8.67079572e+01, 8.73362758e+01, 9.42477796e+01, 1.01787602e+02, 1.03672558e+02, 1.04929195e+02, 1.05557513e+02, 1.06814150e+02, 1.09327424e+02, 1.10584061e+02, 1.11840698e+02, 1.14353973e+02, 1.14982291e+02, 1.16238928e+02, 1.20637158e+02, 1.21893795e+02, 1.22522113e+02 ]

  sinusoidal_coefficients: 
  - [-2.62848330e-01, 0.00000000e+00 ]
  - [-1.34703713e+00, 0.00000000e+00 ]
  - [-1.23555735e+00, 0.00000000e+00 ]
  - [-1.21627201e+00, 0.00000000e+00 ]
  - [1.61445467e+00, 0.00000000e+00 ]
  - [-1.41374468e+00, 0.00000000e+00 ]
  - [7.40048378e-01, 0.00000000e+00 ]
  - [5.53081706e-01, 0.00000000e+00 ]
  - [-1.21414678e+00, 0.00000000e+00 ]
  - [-1.05838811e+00, 0.00000000e+00 ]
  - [-1.90989286e-01, 0.00000000e+00 ]
  - [-9.76809026e-01, 0.00000000e+00 ]
  - [1.17741351e+00, 0.00000000e+00 ]
  - [1.35391301e+00, 0.00000000e+00 ]
  - [-1.50816235e+00, 0.00000000e+00 ]
  - [-2.05954002e-01, 0.00000000e+00 ]
  - [-1.57931345e+00, 0.00000000e+00 ]
  - [-1.03040866e+00, 0.00000000e+00 ]
  - [-1.62704544e+00, 0.00000000e+00 ]
  - [-5.90222344e-01, 0.00000000e+00 ]
  - [-1.40350261e+00, 0.00000000e+00 ]
  - [-1.63042449e+00, 0.00000000e+00 ]
  - [1.63648919e+00, 0.00000000e+00 ]
  - [9.32666746e-01, 0.00000000e+00 ]
  - [-1.61618326e+00, 0.00000000e+00 ]
  - [-1.57714117e+00, 0.00000000e+00 ]
  - [-1.56252353e+00, 0.00000000e+00 ]
  - [1.62004941e+00, 0.00000000e+00 ]
  - [-8.02533844e-01, 0.00000000e+00 ]
  - [3.27309158e-01, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [-4.90256083e+00, 0.00000000e+00 ]
  - [9.29373191e-01, 0.00000000e+00 ]
  - [1.07314565e+00, 0.00000000e+00 ]
  - [-1.09495477e+00, 0.00000000e+00 ]
  - [2.67917280e-01, 0.00000000e+00 ]
  - [8.24360077e-01, 0.00000000e+00 ]
  - [-1.45964788e+00, 0.00000000e+00 ]
  - [-1.54024160e+00, 0.00000000e+00 ]
  - [-1.09731087e+00, 0.00000000e+00 ]
  - [-1.24822200e+00, 0.00000000e+00 ]
  - [-1.62535124e+00, 0.00000000e+00 ]
  - [-1.31304519e+00, 0.00000000e+00 ]
  - [-1.13663582e+00, 0.00000000e+00 ]
  - [-9.19327525e-01, 0.00000000e+00 ]
  - [-6.35365928e-01, 0.00000000e+00 ]
  - [1.62352287e+00, 0.00000000e+00 ]
  - [-4.28966865e-01, 0.00000000e+00 ]
  - [1.27141714e+00, 0.00000000e+00 ]
  - [-1.75973554e-01, 0.00000000e+00 ]
  - [-1.52639482e+00, 0.00000000e+00 ]
  - [8.41679260e-01, 0.00000000e+00 ]
  - [1.41278168e-01, 0.00000000e+00 ]
  - [1.21104958e-02, 0.00000000e+00 ]
  - [-1.34475882e+00, 0.00000000e+00 ]
  - [-2.57284292e-01, 0.00000000e+00 ]
  - [4.36885875e-01, 0.00000000e+00 ]
  - [4.86583767e-01, 0.00000000e+00 ]
  - [-2.31696919e-01, 0.00000000e+00 ]
  - [-1.42624787e+00, 0.00000000e+00 ]
  - [1.60346882e+00, 0.00000000e+00 ]

deflection_6dof_jnt1_test4_:
  signal_ffw: 0
  topic_name: "/rigid/joint_feedforward"
  test_duration: 100.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-01, 9.55044167e+01, 9.73893723e+01, 9.80176908e+01, 9.92743279e+01, 9.99026464e+01, 1.02415921e+02, 1.03044239e+02, 1.04300876e+02, 1.07442469e+02, 1.11212380e+02, 1.12469017e+02, 1.13097336e+02, 1.13725654e+02, 1.16867247e+02, 1.23778751e+02, 1.26292025e+02, 1.34460166e+02, 1.35088484e+02, 1.37601758e+02, 1.40115032e+02, 1.42628306e+02, 1.48283173e+02, 1.49539810e+02, 1.50796447e+02, 1.52053084e+02, 1.53309721e+02, 1.53938040e+02, 1.55194677e+02, 1.55822996e+02 ]

  sinusoidal_coefficients: 
  - [-3.59220797e+00, 0.00000000e+00 ]
  - [4.33067484e-01, 0.00000000e+00 ]
  - [-3.69774217e-01, 0.00000000e+00 ]
  - [1.72345041e-01, 0.00000000e+00 ]
  - [1.19200345e-01, 0.00000000e+00 ]
  - [1.05695544e+00, 0.00000000e+00 ]
  - [-1.44214009e+00, 0.00000000e+00 ]
  - [-5.51764300e-01, 0.00000000e+00 ]
  - [-1.00908845e+00, 0.00000000e+00 ]
  - [-6.61166042e-01, 0.00000000e+00 ]
  - [-1.50841495e+00, 0.00000000e+00 ]
  - [-6.37889849e-02, 0.00000000e+00 ]
  - [-1.25218963e+00, 0.00000000e+00 ]
  - [1.47019339e+00, 0.00000000e+00 ]
  - [2.41606928e-01, 0.00000000e+00 ]
  - [9.13009664e-01, 0.00000000e+00 ]
  - [-4.73613122e-01, 0.00000000e+00 ]
  - [1.15745934e+00, 0.00000000e+00 ]
  - [-1.22528675e+00, 0.00000000e+00 ]
  - [-2.04938090e-01, 0.00000000e+00 ]
  - [1.46494346e+00, 0.00000000e+00 ]
  - [-1.33284284e+00, 0.00000000e+00 ]
  - [4.47344097e-01, 0.00000000e+00 ]
  - [-1.15081458e+00, 0.00000000e+00 ]
  - [1.34380033e+00, 0.00000000e+00 ]
  - [1.48423728e+00, 0.00000000e+00 ]
  - [-9.56451643e-01, 0.00000000e+00 ]
  - [-1.90273890e-01, 0.00000000e+00 ]
  - [1.56139221e+00, 0.00000000e+00 ]
  - [-5.67819385e-01, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [-3.02225689e+00, 0.00000000e+00 ]
  - [-1.50370165e+00, 0.00000000e+00 ]
  - [-1.52050423e+00, 0.00000000e+00 ]
  - [-1.55530167e+00, 0.00000000e+00 ]
  - [1.56027477e+00, 0.00000000e+00 ]
  - [1.15391130e+00, 0.00000000e+00 ]
  - [6.07369790e-01, 0.00000000e+00 ]
  - [1.46431631e+00, 0.00000000e+00 ]
  - [-1.19599607e+00, 0.00000000e+00 ]
  - [1.41828261e+00, 0.00000000e+00 ]
  - [4.16353727e-01, 0.00000000e+00 ]
  - [1.56352072e+00, 0.00000000e+00 ]
  - [9.38449366e-01, 0.00000000e+00 ]
  - [-5.35908088e-01, 0.00000000e+00 ]
  - [1.54605698e+00, 0.00000000e+00 ]
  - [-1.27085776e+00, 0.00000000e+00 ]
  - [-1.49142774e+00, 0.00000000e+00 ]
  - [1.05306883e+00, 0.00000000e+00 ]
  - [-9.73313134e-01, 0.00000000e+00 ]
  - [-1.55134344e+00, 0.00000000e+00 ]
  - [5.50097042e-01, 0.00000000e+00 ]
  - [-8.19875634e-01, 0.00000000e+00 ]
  - [-1.49951637e+00, 0.00000000e+00 ]
  - [-1.06032632e+00, 0.00000000e+00 ]
  - [8.01789723e-01, 0.00000000e+00 ]
  - [4.95687184e-01, 0.00000000e+00 ]
  - [-1.23849358e+00, 0.00000000e+00 ]
  - [1.55321020e+00, 0.00000000e+00 ]
  - [1.03539642e-01, 0.00000000e+00 ]
  - [-1.45816571e+00, 0.00000000e+00 ]

deflection_6dof_jnt1_test5_:
  signal_ffw: 0
  topic_name: "/rigid/joint_feedforward"
  test_duration: 100.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-01, 1.30690254e+02, 1.31946891e+02, 1.32575210e+02, 1.33831847e+02, 1.38858395e+02, 1.39486714e+02, 1.41371669e+02, 1.41999988e+02, 1.42628306e+02, 1.45141581e+02, 1.46398218e+02, 1.48283173e+02, 1.50168129e+02, 1.51424766e+02, 1.53938040e+02, 1.55822996e+02, 1.56451314e+02, 1.57707951e+02, 1.58336270e+02, 1.60849544e+02, 1.62106181e+02, 1.67761048e+02, 1.69017685e+02, 1.74044233e+02, 1.75929189e+02, 1.78442463e+02, 1.79070781e+02, 1.83469011e+02, 1.87867241e+02 ]

  sinusoidal_coefficients: 
  - [4.66330575e+00, 0.00000000e+00 ]
  - [1.27106420e+00, 0.00000000e+00 ]
  - [-1.16893717e+00, 0.00000000e+00 ]
  - [9.08892626e-01, 0.00000000e+00 ]
  - [-1.55061024e+00, 0.00000000e+00 ]
  - [1.52319333e+00, 0.00000000e+00 ]
  - [1.06610782e+00, 0.00000000e+00 ]
  - [-1.34205500e+00, 0.00000000e+00 ]
  - [-1.54858891e+00, 0.00000000e+00 ]
  - [-1.54725604e+00, 0.00000000e+00 ]
  - [8.72300930e-01, 0.00000000e+00 ]
  - [-1.50776495e+00, 0.00000000e+00 ]
  - [3.56806923e-01, 0.00000000e+00 ]
  - [8.67477850e-01, 0.00000000e+00 ]
  - [-7.88174177e-01, 0.00000000e+00 ]
  - [-1.40337576e+00, 0.00000000e+00 ]
  - [1.23161631e+00, 0.00000000e+00 ]
  - [-1.15367065e+00, 0.00000000e+00 ]
  - [-2.94986467e-01, 0.00000000e+00 ]
  - [8.78872480e-01, 0.00000000e+00 ]
  - [1.00657384e+00, 0.00000000e+00 ]
  - [-1.54323733e+00, 0.00000000e+00 ]
  - [1.49328948e+00, 0.00000000e+00 ]
  - [-1.09333085e+00, 0.00000000e+00 ]
  - [1.27437243e+00, 0.00000000e+00 ]
  - [-4.75514381e-01, 0.00000000e+00 ]
  - [-4.44629167e-01, 0.00000000e+00 ]
  - [-1.30755732e+00, 0.00000000e+00 ]
  - [-1.36003419e+00, 0.00000000e+00 ]
  - [9.81440106e-01, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [-1.02080192e-01, 0.00000000e+00 ]
  - [8.95445458e-01, 0.00000000e+00 ]
  - [-1.02518909e+00, 0.00000000e+00 ]
  - [-1.26148364e+00, 0.00000000e+00 ]
  - [1.14169398e-01, 0.00000000e+00 ]
  - [3.11943649e-01, 0.00000000e+00 ]
  - [-1.13174241e+00, 0.00000000e+00 ]
  - [-7.85057414e-01, 0.00000000e+00 ]
  - [-1.38921383e-01, 0.00000000e+00 ]
  - [-1.53053998e-01, 0.00000000e+00 ]
  - [-1.28705783e+00, 0.00000000e+00 ]
  - [3.79567679e-01, 0.00000000e+00 ]
  - [1.51331279e+00, 0.00000000e+00 ]
  - [-1.29031351e+00, 0.00000000e+00 ]
  - [1.34022693e+00, 0.00000000e+00 ]
  - [-6.69300559e-01, 0.00000000e+00 ]
  - [-9.48972087e-01, 0.00000000e+00 ]
  - [-1.04233910e+00, 0.00000000e+00 ]
  - [-1.52656797e+00, 0.00000000e+00 ]
  - [-1.28257941e+00, 0.00000000e+00 ]
  - [1.18500459e+00, 0.00000000e+00 ]
  - [-1.89328607e-01, 0.00000000e+00 ]
  - [4.33028061e-01, 0.00000000e+00 ]
  - [-1.10546570e+00, 0.00000000e+00 ]
  - [-8.90730977e-01, 0.00000000e+00 ]
  - [-1.48030836e+00, 0.00000000e+00 ]
  - [1.48987640e+00, 0.00000000e+00 ]
  - [-8.41261333e-01, 0.00000000e+00 ]
  - [-7.53481104e-01, 0.00000000e+00 ]
  - [1.20590302e+00, 0.00000000e+00 ]

deflection_6dof_jnt1_test6_:
  signal_ffw: 0
  topic_name: "/rigid/joint_feedforward"
  test_duration: 100.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-01, 1.62106181e+02, 1.66504411e+02, 1.70902640e+02, 1.72159277e+02, 1.72787596e+02, 1.76557507e+02, 1.96035382e+02, 1.99805293e+02, 2.01690248e+02, 2.05460160e+02, 2.09230071e+02, 2.10486708e+02, 2.11115026e+02, 2.12371663e+02, 2.12999982e+02, 2.14884938e+02, 2.19283167e+02, 2.20539804e+02, 2.24309715e+02, 2.28079627e+02, 2.28707945e+02, 2.29336264e+02, 2.31849538e+02, 2.33106175e+02, 2.33734493e+02, 2.37504405e+02, 2.38132723e+02, 2.41902634e+02, 2.50070775e+02 ]

  sinusoidal_coefficients: 
  - [-3.58133689e+00, 0.00000000e+00 ]
  - [6.67495670e-01, 0.00000000e+00 ]
  - [-9.11239664e-01, 0.00000000e+00 ]
  - [-1.41595008e+00, 0.00000000e+00 ]
  - [1.42822014e+00, 0.00000000e+00 ]
  - [-1.09353246e+00, 0.00000000e+00 ]
  - [1.77864234e-01, 0.00000000e+00 ]
  - [-1.39779453e+00, 0.00000000e+00 ]
  - [1.56953689e-01, 0.00000000e+00 ]
  - [-1.23304063e+00, 0.00000000e+00 ]
  - [1.26596458e+00, 0.00000000e+00 ]
  - [-1.45678610e+00, 0.00000000e+00 ]
  - [1.07986489e+00, 0.00000000e+00 ]
  - [-6.32767197e-01, 0.00000000e+00 ]
  - [-4.16296673e-01, 0.00000000e+00 ]
  - [-1.05671961e+00, 0.00000000e+00 ]
  - [5.75540918e-01, 0.00000000e+00 ]
  - [1.44063195e+00, 0.00000000e+00 ]
  - [6.54663638e-01, 0.00000000e+00 ]
  - [1.71895708e-01, 0.00000000e+00 ]
  - [-3.62658901e-01, 0.00000000e+00 ]
  - [-8.70794421e-01, 0.00000000e+00 ]
  - [-1.33550618e+00, 0.00000000e+00 ]
  - [5.78943798e-01, 0.00000000e+00 ]
  - [8.53518055e-01, 0.00000000e+00 ]
  - [8.27850603e-01, 0.00000000e+00 ]
  - [9.96302540e-01, 0.00000000e+00 ]
  - [-1.17754046e+00, 0.00000000e+00 ]
  - [-5.80133429e-01, 0.00000000e+00 ]
  - [1.40763295e+00, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [2.71492658e+00, 0.00000000e+00 ]
  - [-1.34109599e+00, 0.00000000e+00 ]
  - [-1.18900429e+00, 0.00000000e+00 ]
  - [4.89054483e-01, 0.00000000e+00 ]
  - [-4.51969193e-01, 0.00000000e+00 ]
  - [1.02385335e+00, 0.00000000e+00 ]
  - [-1.48743176e+00, 0.00000000e+00 ]
  - [-5.38757248e-01, 0.00000000e+00 ]
  - [-1.48978336e+00, 0.00000000e+00 ]
  - [-8.50705426e-01, 0.00000000e+00 ]
  - [-8.00888631e-01, 0.00000000e+00 ]
  - [3.49089090e-01, 0.00000000e+00 ]
  - [1.03825851e+00, 0.00000000e+00 ]
  - [1.35782716e+00, 0.00000000e+00 ]
  - [-1.43902259e+00, 0.00000000e+00 ]
  - [1.06180629e+00, 0.00000000e+00 ]
  - [-1.38305516e+00, 0.00000000e+00 ]
  - [-4.10692706e-01, 0.00000000e+00 ]
  - [-1.34740656e+00, 0.00000000e+00 ]
  - [-1.48813332e+00, 0.00000000e+00 ]
  - [1.45346739e+00, 0.00000000e+00 ]
  - [-1.21893642e+00, 0.00000000e+00 ]
  - [-6.78610466e-01, 0.00000000e+00 ]
  - [1.38163418e+00, 0.00000000e+00 ]
  - [-1.23109539e+00, 0.00000000e+00 ]
  - [1.24850002e+00, 0.00000000e+00 ]
  - [1.11869127e+00, 0.00000000e+00 ]
  - [9.26006146e-01, 0.00000000e+00 ]
  - [1.38113509e+00, 0.00000000e+00 ]
  - [-5.12502106e-01, 0.00000000e+00 ]

deflection_6dof_jnt1_test7_:
  signal_ffw: 0
  topic_name: "/rigid/joint_feedforward"
  test_duration: 100.000000
  start_stop_transient: 0.1
  natural_frequency: [6.28318531e-01, 1.96663700e+02, 2.08601752e+02, 2.13628300e+02, 2.17398212e+02, 2.29336264e+02, 2.33106175e+02, 2.41902634e+02, 2.45672546e+02, 2.46300864e+02, 2.51955731e+02, 2.58238916e+02, 2.58867235e+02, 2.60752190e+02, 2.63265464e+02, 2.67663694e+02, 2.68920331e+02, 2.78345109e+02, 2.78973428e+02, 2.79601746e+02, 2.83371657e+02, 2.87769887e+02, 2.93424754e+02, 2.95938028e+02, 2.97194665e+02, 3.00336258e+02, 3.02849532e+02, 3.03477850e+02, 3.05991124e+02, 3.07876080e+02 ]

  sinusoidal_coefficients: 
  - [-9.93948623e-01, 0.00000000e+00 ]
  - [1.52489255e+00, 0.00000000e+00 ]
  - [-1.73199333e-01, 0.00000000e+00 ]
  - [-1.14250032e+00, 0.00000000e+00 ]
  - [-2.73215677e-01, 0.00000000e+00 ]
  - [-1.12714670e+00, 0.00000000e+00 ]
  - [4.98171897e-01, 0.00000000e+00 ]
  - [1.52360828e+00, 0.00000000e+00 ]
  - [-1.62419097e-01, 0.00000000e+00 ]
  - [5.26924264e-01, 0.00000000e+00 ]
  - [1.49321330e+00, 0.00000000e+00 ]
  - [-7.13194430e-01, 0.00000000e+00 ]
  - [-8.79204664e-01, 0.00000000e+00 ]
  - [8.72086066e-01, 0.00000000e+00 ]
  - [4.19053758e-01, 0.00000000e+00 ]
  - [8.58472039e-02, 0.00000000e+00 ]
  - [-1.69475849e-01, 0.00000000e+00 ]
  - [-9.07335764e-01, 0.00000000e+00 ]
  - [4.47264515e-01, 0.00000000e+00 ]
  - [2.02239090e-02, 0.00000000e+00 ]
  - [-7.50306198e-01, 0.00000000e+00 ]
  - [6.05665595e-01, 0.00000000e+00 ]
  - [-1.46993455e+00, 0.00000000e+00 ]
  - [-1.24540923e+00, 0.00000000e+00 ]
  - [1.08272153e+00, 0.00000000e+00 ]
  - [8.52596423e-01, 0.00000000e+00 ]
  - [1.10879040e-01, 0.00000000e+00 ]
  - [1.40795427e+00, 0.00000000e+00 ]
  - [-1.52377609e+00, 0.00000000e+00 ]
  - [1.18118041e+00, 0.00000000e+00 ]

  cosinusoidal_coefficients: 
  - [4.48719622e+00, 0.00000000e+00 ]
  - [1.47266973e-01, 0.00000000e+00 ]
  - [1.52216519e+00, 0.00000000e+00 ]
  - [-1.02062622e+00, 0.00000000e+00 ]
  - [-1.50742763e+00, 0.00000000e+00 ]
  - [-1.03755732e+00, 0.00000000e+00 ]
  - [-1.44872690e+00, 0.00000000e+00 ]
  - [-1.60008322e-01, 0.00000000e+00 ]
  - [-1.52335318e+00, 0.00000000e+00 ]
  - [1.43851857e+00, 0.00000000e+00 ]
  - [3.42489271e-01, 0.00000000e+00 ]
  - [1.35585345e+00, 0.00000000e+00 ]
  - [-1.25458520e+00, 0.00000000e+00 ]
  - [1.25954387e+00, 0.00000000e+00 ]
  - [1.47355991e+00, 0.00000000e+00 ]
  - [-1.52958005e+00, 0.00000000e+00 ]
  - [-1.52258425e+00, 0.00000000e+00 ]
  - [-1.23439324e+00, 0.00000000e+00 ]
  - [1.46524377e+00, 0.00000000e+00 ]
  - [-1.53185373e+00, 0.00000000e+00 ]
  - [1.33567416e+00, 0.00000000e+00 ]
  - [1.40717947e+00, 0.00000000e+00 ]
  - [4.31598507e-01, 0.00000000e+00 ]
  - [8.92155101e-01, 0.00000000e+00 ]
  - [-1.08383529e+00, 0.00000000e+00 ]
  - [1.27281743e+00, 0.00000000e+00 ]
  - [-1.52796947e+00, 0.00000000e+00 ]
  - [-6.03862269e-01, 0.00000000e+00 ]
  - [-1.58402320e-01, 0.00000000e+00 ]
  - [-9.75601194e-01, 0.00000000e+00 ]

